update-annotations-table.R

```diff
@@ -45,23 +85,23 @@ create_rows <- function(file) {
   ## If term has enumerated values in anyOf, fromJSON will create a handy data
   ## frame of those. We use that in combination with return_df to create a row
   ## for each value.
+  ## If term references the enumerated values of another term, grab those. Term
+  ## cannot extend the values in a referenced schema. Schema referenced should
+  ## exist in current set of.
```

Comment:
I think this might be missing some text at the end of the sentence

Reply:
My train of thought apparently stopped right there. 😆 Thanks for always checking the comments! I'll update that later.